diff --git a/dist/index.d.ts b/dist/index.d.ts
index d33674c95675a5a1818f16dbf67ffca7f3d4791f..51fbdbd3d7d8bec2401874ad556d191e54821d56 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -195,7 +195,7 @@ declare class Table {
 	): void;
 }
 
-declare class EntityCommands extends BaseEntity {
+export declare class EntityCommands extends BaseEntity {
 	#private;
 	constructor(commands: Commands, id: bigint);
 	get id(): bigint;
@@ -207,7 +207,7 @@ type Command = {
 	dataStart: number;
 	dataSize: number;
 };
-declare class Commands {
+export declare class Commands {
 	#private;
 	static fromWorld(world: World): Commands;
 	constructor(world: World, initialValuePointers: number[], pointer: number);
@@ -604,7 +604,7 @@ type Descriptors = typeof descriptors;
 
 type SystemFunction<T extends any[]> = (...args: T) => any;
 type ParameterCreator = (descriptors: Descriptors) => Descriptor[];
-type SystemDependencies = {
+export type SystemDependencies = {
 	dependencies: SystemDefinition[];
 	implicitPosition: -1 | 0 | 1;
 };